sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -225,6 +225,32 @@ Table {counter: tableNumber}. Terminology.
 
 |===
 
+==== Event Terminology  [[event-terminology]]
+
+:events-spec: https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#sec:interface.event
+
+For the purposes of clarity when talking about events in this specification we
+will split events into two categories:
+
+- *Limited graph events*: These are events returned from a queue submission
```

Comment:
How about to rename "limited graph events" to "graph limited events", to say that the events are limited to be valid within a graph scope?

"Limited graph events" gives me the wrong impression that there is another type of "full graph events",

Reply:
I've updated the usage to "graph-limited events".